audit-base/src/main/java/com/wultra/core/audit/base/util/ClassUtil.java

```diff
@@ -24,21 +26,27 @@ public class ClassUtil extends SecurityManager {
 
     /**
      * Get calling class.
-     * @param packageFilter Package to filter out when resolving calling class.
+     * @param packageFilter Packages to filter out when resolving calling class.
      * @return Calling class.
      */
-    public static Class<?> getCallingClass(String packageFilter) {
+    public static Class<?> getCallingClass(List<String> packageFilter) {
         final Class<?>[] trace = INSTANCE.getClassContext();
         if (trace == null) {
             return null;
         }
+        classLoop:
```

Comment:
Not a big fan of this... could we do it without the label?

Reply:
Btw this is one of the few examples where "Stream API" would work nicely for me.

Reply:
Sure, why not, I will change it.